helpme/views/helpme_admin.py

```diff
@@ -84,20 +85,20 @@ def get_context_data(self, **kwargs):
         return context
 
     def form_valid(self, form):
-        action = " updated the "
+        action = _(" updated the ")
```

Comment:
Concatenating strings makes an assumption that all languages follow English Grammar structure.  The solution is to capture all the values in a dictionary and then use key replacement with a localizable formatted string at the end.

Reply:
Safer option is to have the data in a JSON string that we can use in a trans tag in the template.